.github/workflows/deploy.yml

```diff
@@ -183,6 +183,14 @@ jobs:
         id: setup-node
         uses: ./.github/actions/composite/setupNode
 
+     - name: Install node modules
+        run: |
+          npm install
+          cd .. && npm install
```

Comment:
I think `npm install` was run in `setup-node` step above. So only `npm run grunt:build:shared` should be run